qualtran/cirq_interop/_cirq_to_bloq.py

```diff
@@ -119,6 +119,9 @@ def as_cirq_op(
         qubits = in_quregs.get('q', ()).flatten()
         return self.cirq_gate.on(*qubits), in_quregs
 
+    def adjoint(self) -> 'Bloq':
```

Comment:
Do we have a test for cirq-bloq interop with adjoint? If not, I think there should be a unit test tests the unitary / simulated result is correct.

Reply:
The adjoint protocol will break on any bloq that has an attribute named `adjoint`. For reviewing convenience, I split those changes into #566

Reply:
OH, there's an And here, got you